client/dist/styles/questionsAnswers.css

```diff
@@ -308,7 +227,119 @@ p.answerHelpfulText, p.userHelpfulBtn, p.userReportBtn{
   margin-right: 2.5em;
   justify-content: start;
 }
+
+/* questionAdd form CSS */
+.addQuestionAsterisk.productTitle, .addQuestionAsterisk.userNickName, .addQuestionAsterisk.userEmail {
+  margin: 1vh;
+}
+.addQuestionAsterisk.productTitle:after, .addQuestionAsterisk.userNickName:after, .addQuestionAsterisk.userEmail:after{
+  content: ' *';
+  color: #B22222;
+  height: 3vh;
+}
+.qFormData {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  background-color: lightsteelblue;
+  border-radius: 1em;
+  margin-bottom: 1vh;
+  padding:.7rem;
+  z-index: 2;
+  height: 75vh;
+  width: 35vw;
+}
+.qFormHide {
+  display:  none;
+}
+.qForm {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  z-index: 1;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+.questionClose.container {
+  display: flex;
+  flex-direction: column;
+  height: 5vh;
+  width: 15vw;
+}
+.questionCloseMarker {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  height: 1vh;
+  width: 1vw;
+}
+.questionCloseMarker:hover{
+  color: darkred;
```

Comment:
Ahh interesting to change color! I set my small buttons to grow slightly on hover to make them more obviously buttons. For my remove from outfit button, I might add color change to red as well